src/components/Tree/child.js

```diff
@@ -30,14 +32,20 @@ export default function Child(props) {
         name,
         selectedNode,
         isFirstNode,
+        autoFocus,
+        focusedNode,
+        setFocusedNode,
+        clearFocusedNode,
+        privateKeyDown,
```

Comment:
we should follow the convention to name the callbacks starting with the on prefix like:
onPrivateKeyDown, onFocusNode, ...